packages/mobile/src/web3/dataEncryptionKey.ts

```diff
@@ -118,23 +189,129 @@ export function* registerAccountDek(account: string) {
   }
 }
 
+// Unlike normal DEK registration, registration via Komenci should be considered fatal. If there
+// is no on-chain mapping of accountAddresss => walletAddress, then senders will erroneously
+// send to MTW instead of EOA. A no-op if registration has already been done
+export function* registerWalletAndDekViaKomenci(
+  komenciKit: KomenciKit,
+  accountAddress: string,
+  walletAddress: string
+) {
+  ValoraAnalytics.track(OnboardingEvents.account_dek_register_start)
```

Comment:
Adding a property to this event noting that this is a Komenci assisted registration would be helpful